commands.js

```diff
@@ -0,0 +1,337 @@
+var formulas = require("./formulas.js");
```

Comment:
You're defining `formulas` here, while you're using `formula` below. That causes the bot to error before even booting up.

Reply:
I fixed the typos